jwql/jwql_monitors/generate_preview_images.py

```diff
@@ -577,7 +576,10 @@ def group_filenames(filenames):
         subgroup = []
 
         # Generate string to be matched with other filenames
-        filename_dict = filename_parser(os.path.basename(filename))
+        try:
+            filename_dict = filename_parser(os.path.basename(filename))
+        except ValueError:
+            break
```

Comment:
Should we add a note to the log if the filename parser fails rather than silently skipping that file? It might be nice to see what files aren't being parsed correctly.

Reply:
Great idea!  I will add that in before I merge.